package.json

```diff
@@ -12,38 +12,38 @@
     "yarn": ">=0.25.2"
   },
   "dependencies": {
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
```

Comment:
Why the hard version dependencies?

Reply:
Sorry about this probably sneaked in while I was trying to update some packages between different branches.